### DISCUSSION TITLE
Unable to find "Mine" translation on crowdin

### DISCUSSION THREAD

**Discussion by WEBudoGT** [chatwoot/chatwoot#3828] [Translations]
We're trying to translate as much as we can, but one of the issues we found is that the term "Mine" (used on the conversation tabs) is not referenced anywhere and it's horribly translated.
Mine is being translated as an explosive device (Mina) in spanish, instead of the possesive term which means "I own" (Mio).
In this case it should say "Mías" (femenine of "my own" as "conversations" in spanish is referred in femenine form (las conversaciones)).
Well... I know it's hard to explain and for you to understand the differences, but I need to know where can I change this, or if this is an "in-code" translation, please change it.
Also the tab "Todos" is wrong, because it should say "Todas" (again, because conversations is femenine term in spanish).
"Todas las conversaciones"

Please advise

Thanks!

**Comment by sojan-official** [comment]
@WEBudoGT let me take a look a this and get back.

**Reply by sojan-official** [reply]
@WEBudoGT the reason why the string doesn't show up is that the string already went through the crowdsourcing step and was marked as done via proofreading.  After which you would need proofreading permissions to edit the string. 

- I have fixed this string for now with 'Mías' on crowdin.  If you are interested in contributing more, Could you ping me on crowdin, we will add you as a proofreader for Spanish

**Comment by grupale** [comment]
Hi there! I've just ended the suggestions for the missing translations. When you approve it, I'll need to do an update? I have a docker installation.
![image](https://user-images.githubusercontent.com/10934794/173978235-792c1dd0-4d2e-4f7f-921e-7e2e63db65f2.png)

I couldn't translate the Unassigned string from the Inbox section. I didn't find the string in Crowdin. Can you tell me where to find it please?
![image](https://user-images.githubusercontent.com/10934794/173978253-953b2484-20c0-4a81-bae0-5bf8e63f9f67.png)
![image](https://user-images.githubusercontent.com/10934794/173978263-2cec0fca-321a-472c-ae67-52751933a154.png)
Thanks!

**Reply by vishnu-narayanan** [reply]
@grupale Translations from Crowdin are merged to `develop` branch periodically. Chatwoot follows a monthly release cycle in which these changes would be merged to `master` and the release is cut. This is followed by releasing official docker images having the latest changes. 

Updating a docker installation using the official image would just be changing the docker image tag to the newest version. 

Also, if you are in a hurry you could always decide to build the image from your end and use it with your deployment.
